src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: $white-accent;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $width-card;
+  padding: 32px 16px 16px;
+  margin: $card-margin-top $card-margin-right 0 0;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-child(-n + 4) {
+    margin-top: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+  
+  &__name-product {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 1.5;
+    color: $main-accent;
+    margin: 0 auto 4px;
+  }
+  
+  &__code {
+    @extend %small-text;
+    color: $secondary-accent;
+    margin-bottom: 16px;
+  }
+  
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+  
+  &__review {
+    @extend %small-text;
+    color: $main-accent;
+  }
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+  
+  &__text {
+    font-size: 12px;
+    line-height: 1.5;
+    color: $secondary-accent;
+  }
+  
+  &__dollar {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 1.1;
+    color: $main-accent;
+  }
+  
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    background: $blue-accent;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 1.1;
+    text-align: center;
```

Comment:
text align redundant here

Reply:
Thank you for your approval. I have already fixed your comments